priv/scripts/elixir.sh

```diff
@@ -108,6 +108,13 @@ function upload_docs {
     pushd ${app}
     gsutil -m -h "cache-control: public,max-age=3600" -h "x-goog-meta-surrogate-key: docspage/${app}/${version}" -h "x-goog-meta-surrogate-control: public,max-age=604800" rsync -d -r . "gs://hexdocs.pm/${app}/${version}"
     fastly_purge $BOB_FASTLY_SERVICE_HEXDOCS "docspage/${app}/${version}"
+
+    mkdir tmp
+    mv docs_config.js tmp/
+    gsutil -m -h "cache-control: public,max-age=3600" -h "x-goog-meta-surrogate-key: docspage/${app}/docs_config.js" -h "x-goog-meta-surrogate-control: public,max-age=604800" rsync tmp "gs://hexdocs.pm/${app}"
```

Comment:
How come `rsync` instead of `cp`?

Reply:
oh if that's gonna work even better. I was assuming rsync was for that gs:// url, will try this!

Reply:
I think any command that accepts a local path will also accept a `gs://` URL.